skymp5-functions-lib/src/logic/SweetPieGameModeListener.ts

```diff
@@ -110,14 +111,41 @@ export class SweetPieGameModeListener implements GameModeListener {
             random.push(`${Math.floor(Math.random() * (max) + 1)}`);
           }
         }
-        let message: string;
+        let message: ChatMessage = {
+          opacity: 1,
+          sender: {
+            masterApiId: 0,
+            gameId: "0",
+          },
+          category: 'system',
+          text: []
+        };;
         if (max === 2) {
-          message = `#{${colors[max] ? colors[max] : '9159B6'}}${senderName} подбрасывает монетку #{FFFFFF}- ${random.join(', ')}`;
+          message.text = [
+            {
+              text: `${senderName} подбрасывает монетку`,
+              color: colors[max] ? colors[max] : '9159B6'
+            },
+            {
+              text: `- ${random.join(', ')}`,
+              color: 'FFFFFF'
```

Comment:
I think better to use full `#FFFFFF` here. In case we want to specify some interesting colors like `rgba(r, g, b, a)` or human-readable built-in colors here. But idk